src/renderer/components/dex/TradeHistory.vue

```diff
@@ -43,9 +41,6 @@ export default {
 
   mounted() {
     this.loadTrades();
-    loadTradesIntervalId = setInterval(() => {
```

Comment:
Can’t remove this yet actually unless we handle reloading transactions on websocket connect and reconnect.

Reply:
Will add this logic to SOCKET_RECONNECT